src/mainScanner.py

```diff
@@ -19,7 +19,7 @@ def homepage():
     return render_template('index.html')
 
 
-exc = getattr(__builtin__, "IOError", "FileNotFoundError")
+exc = getattr(__builtins__, "IOError", "FileNotFoundError")

```

Comment:
Not sure if this line would not throw error while running.
It should be 
`exc = getattr(builtins, "IOError", "FileNotFoundError")`
instead of 
`exc = getattr(__builtins__, "IOError", "FileNotFoundError")`,
I suppose.

[Reference.](https://github.com/vinitshahdeo/PortScanner/blob/feature/python3.8/src/mainScanner.py)

@ranaaditya does this file execute normally with `__builtins__` ?

Reply:
Yeah, it was working fine in my local.